src/mscorlib/shared/System/Runtime/CompilerServices/AsyncMethodBuilder.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace System.Runtime.CompilerServices
+{
+    internal static partial class AsyncMethodBuilder
+    {
+        /// <summary>Initiates the builder's execution with the associated state machine.</summary>
+        /// <typeparam name="TStateMachine">Specifies the type of the state machine.</typeparam>
+        /// <param name="stateMachine">The state machine instance, passed by reference.</param>
+        [DebuggerStepThrough]
```

Comment:
Why DebuggerStepThrough?

Reply:
Hmm, nevermind, I see that it's there on the original for some reason.  Not sure why it's there either, but we can leave this for consistency.